modules/monitoring-searching-and-filtering-targets.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="monitoring-searching-and-filtering-targets_{context}"]
+= Searching and filtering metrics targets
+
+The list of metrics targets can be long. You can filter and search these targets based on various criteria.
```

Comment:
the paragraph seems imperfect, I think we can add:
For example, search via Status or Source